app/models/framework/definition.rb

```diff
@@ -2,6 +2,12 @@ class Framework
   module Definition
     class MissingError < StandardError; end
 
+    def self.[](framework_short_name)
+      "Framework::Definition::#{framework_short_name}".constantize
```

Comment:
This will break for CM/OSG/05/3565 right? is the plan to sort that out down the road?

Reply:
No, we should sort that now 😄

Reply:
(I was remarkably slapdash about this in order to get a working interface for @leeky – this needs specs, with that in it, and that case just shows why!)